packages/core/src/errors/InruptError.test.ts

```diff
@@ -75,11 +76,15 @@ describe("InruptError", () => {
       }) as Response & { ok: false };
 
       const message = "Normal error string...";
-      const error = new InruptError(message, undefined, failedResponse, false);
+      const error = new InruptError(message).httpResponse(
+        failedResponse,
+        false
+      );
 
       expect(error.name).toEqual("Error");
-      expect(error.statusCode).toEqual(404);
-      expect(error.statusText).toEqual("Not Found");
+      expect(error.hasHttpResponse()).toBeTruthy();
```

Comment:
Do we want to test for truthiness, or for equality with `true` ?

Reply:
Yep: 2d63c87